### PR TITLE
Add query to check is memcached is disabled in elasticache. Closes #2292

### LINKED ISSUE #2292: Add "Memcached Disabled" query for CloudFormation
### Platform
CloudFormation

### Description
Add support in CloudFormation for the query already implemented in Terraform and Ansible.
Check if the Memcached is disabled on the ElastiCache

### PR DESCRIPTION
**Pull request by joaomartinscx**
Closes #2292

**Proposed Changes**

- Add support in CloudFormation for the query that checks if the engine used in ElastiCache clusters isn't memcached

I submit this contribution under Apache-2.0 license.

### CODE REVIEW

File: assets/queries/cloudFormation/memcached_disabled/test/negative2.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "Description": "A sample template",
+  "Resources": {
+    "ElasticacheCluster": {
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/memcached_disabled/test/negative2.json:4]: Change "ElasticacheCluster" to "ElasticacheCluster2"

File: assets/queries/cloudFormation/memcached_disabled/test/positive1.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: A sample template
+Resources:
+  ElasticacheCluster:
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/memcached_disabled/test/positive1.yaml:4]: Change "ElasticacheCluster" to "ElasticacheCluster3"

File: assets/queries/cloudFormation/memcached_disabled/test/positive2.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "Description": "A sample template",
+  "Resources": {
+    "ElasticacheCluster": {
```
Comment (Ruben-Silva) [inline assets/queries/cloudFormation/memcached_disabled/test/positive2.json:4]: Change "ElasticacheCluster" to "ElasticacheCluster4"

**Review by Ruben-Silva — DISMISSED** [review]

File: assets/queries/cloudFormation/memcached_disabled/test/negative2.json (hunk 1 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/memcached_disabled/test/negative2.json:4]: Why does it need to be change from sample to sample?

Comment (Ruben-Silva) [reply assets/queries/cloudFormation/memcached_disabled/test/negative2.json:4]: I think all the samples of a query are grouped together, at some point, and having the same name can lead to conflicts. At least, I have seen PRs that the only thing they changed was differentiating the resource names, so I thought better safe than sorry.

Comment (joaomartinscx) [reply assets/queries/cloudFormation/memcached_disabled/test/negative2.json:4]: Is this part of the engine? End-users can't submit different files with the same resources' names without the risk of conflicts?
@ruigomescx if you could have an input on this as well

Comment (joaomartinscx) [reply assets/queries/cloudFormation/memcached_disabled/test/negative2.json]: done

File: assets/queries/cloudFormation/memcached_disabled/test/positive2.json (hunk 3 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/memcached_disabled/test/positive2.json]: done

File: assets/queries/cloudFormation/memcached_disabled/test/positive1.yaml (hunk 2 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/memcached_disabled/test/positive1.yaml]: done

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**